client/ayon_resolve/api/constants.py

```diff
@@ -2,6 +2,7 @@
 rename_index = 0
 rename_add = 0
 
+selected_clip_color = "Chocolate"
```

Comment:
I know that you've added only this constant (and you followed what was here), but it would nice to use upper case variable names, for constants.

Reply:
Agreed. And you can also change to uppercase the rest, please.

Reply:
Sure thing.